Add `cds add handler` docs (#1273)
cap-js/docs: tools/cds-cli.md

| `typer`                       |       <X/>       |      <Na/>       |
| `typescript`                  |       <X/>       |      <Na/>       |
| `completion`                  |       <X/>       |       <X/>       |

> | `handler`(#handler)           |       <Na/>      |       <X/>       |

Reviewer: ```suggestion
| [`handler`](#handler)         |       <Na/>      |       <X/>       |
```

For CloudFoundry, use `cf login ...` and select org and space.
:::

### handler <Since version="8.3.0" of="@sap/cds-dk" /> {.add}

Generates handler stubs for actions and functions in Java projects.

> The simplest form of:
> 

Reviewer 2: ```suggestion
Execute the following from the `srv` directory of the project to generates handler files for all actions and functions:
```

```sh
cds add handler
```

> called from the `srv` directory of the project generates handler files for all actions and functions.

Reviewer 2: ```suggestion
```

#### Filtering {#handler-filtering}

> You can filter by the names of actions and functions.

Reviewer 2: That means you can select specific actions/functions for which handlers should be created, right? If so, eventually the title of this section should also be different.

```suggestion
Use the `--filter` option to create handlers for specific actions and functions.
```

Author: What do you have in mind? My thinking was to use `Filtering` as the simplest title for this option

```sh
cds add handler --filter submitOrder
```

> #### Other options
> 
> The `--out` option for specifying custom output directories and the `--force` option for overwriting existing files are available as well.
> 

Reviewer 2: ```suggestion
#### More Options

| Option | Description |
| --- | --- |
| `--out` | Specify custom output directories |
| `--force` | Overwrite existing files |

```

For CloudFoundry, use `cf login ...` and select org and space.
:::

### handler <Since version="8.3.0" of="@sap/cds-dk" /> {.add}

Generates handler stubs for actions and functions in Java projects.

Execute the following from the _srv_ directory of the project to generate handler files for all actions and functions:
```sh
cds add handler
```

> #### Filtering {#handler-filtering}

Reviewer 2: That's a tough one for me. Probably:
```suggestion
#### (Re-)Create Specific Handlers {#handler-filtering}
```

Reviewer: > That's a tough one for me. 

Why that?  Filtering means to select what is to be generated.

> (Re-)Create Specific Handlers

That is confusing to me :)

Reviewer 2: Well I understand that I use filtering to find the action/function. But ultimately I like to create handler stubs, or? That's why I'd change the title to sth that tells the story. But if I'm off the track here, let's keep filtering :)

Author: Mhh your call. I would go with Filtering or Create Specific Handlers, without the Re-.

For CloudFoundry, use `cf login ...` and select org and space.
:::

#### handler <Since version="8.3.0" of="@sap/cds-dk" /> {.add}

Generates handler stubs for actions and functions in Java projects. 

The simplest form of:

```sh
cds add handler
```

called from the `srv` directory of the project generates handler files for all actions and functions.

#### Filtering {#handler-filtering}

You can filter by the names of actions and functions. 

```sh
cds add handler --filter submitOrder
```

#### Other options

The `--out` option for specifying custom output directories and the `--force` option for overwriting existing files are available as well.

> 
> ```sh
> cds add handler --out customFolder --force
> ```

Reviewer: No need for a snippet I'd say.

```suggestion
```
